assets/src/stories-editor/components/post-selector/index.js

```diff
@@ -0,0 +1,339 @@
+/**
```

Comment:
Instead of building a custom component why not reuse the autocomplete used by the font picker? This would keep accessibility and styling inline.

Reply:
Worth noting that development on this was started way before the font picker was implemented and merged.

Also, I don't think that component allows for a dynamic list fetched via an API.

Reply:
I believe you are wrong here. Check this example from gutenberg that you originally sent over. https://github.com/WordPress/gutenberg/pull/7385/files#diff-8267665c4a351e7321fa6763c14922d3R30 This is doing a api requests to get a list of users. 

[Source](https://github.com/alphagov/accessible-autocomplete#source) is a function, you can put whatever logic you want in there, so a call to the api could be put in there.

Reply:
While this is worthwhile to explore, I don't have the capacity to replace the component with the Autocomplete one in time for tomorrow. Would you have time to do that?

Otherwise I suggest exploring this between beta and RC. Thoughts?